geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/StatHelpers.scala

```diff
@@ -0,0 +1,10 @@
+package org.locationtech.geomesa.utils.stats
+
+import java.text.SimpleDateFormat
+
+import org.joda.time.format.DateTimeFormat
+
+object StatHelpers {
+  val javaDateFormat = new SimpleDateFormat("EEE MMM dd HH:mm:ss z yyyy")
```

Comment:
you probably don't want to have this in an object - java data formats aren't thread safe. (joda ones are, so the below line is ok)